DEVELOPMENT.md

```diff
@@ -40,6 +40,9 @@ Then, the test commands are as follows:
 - Playwright: `yarn test:playwright`, then `yarn archive-storybook:playwright` to see the archived UI
 - Cypress: `yarn test:cypress`, then `yarn archive-storybook:cypress` to see the archived UI
 
+> [!IMPORTANT]
+> Playwright may require you to run `yarn playwright install` first before any of its e2e tests will pass.
```

Comment:
You could also add the command in the `Getting Started` section.

Reply:
I like that better. Updated.